### ISSUE TITLE
A language bug in the latest MuseScore version

### ISSUE THREAD

**Issue by YBPe** [musescore/MuseScore#24815]
### Issue type

Other type of issue

### Description with steps to reproduce

1. Open the software in Hebrew. 2. Click on the "new score" 3. Some words appear in Hebrew and some in Arabic. 4. I expect all words to appear in Hebrew. 
![MuseScore language bug](https://github.com/user-attachments/assets/9135afc7-1dc6-4bc0-bc25-67bbc7ad86f2)

### Supporting files, videos and screenshots

In addition some of the opptions in the toolbar appear in Hebrew and when pressed changes to Arabic.
![MuseScore language bug2](https://github.com/user-attachments/assets/f76b06fd-88ef-4ac1-9d53-45bd155bab90)

### What is the latest version of MuseScore Studio where this issue is present?

4.4.2

### Regression

Yes, this used to work in a previous version of MuseScore 4.x

### Operating system

Windows 10, 11

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by Jojo-Schmitz** [comment]
Stems from https://musescore.org/en/node/369006 and in https://musescore.org/en/node/369006#comment-1259202 a connection with QT_QUICK_BACKEND being opengl or d3d11, but not with software

**Comment by Jojo-Schmitz** [comment]
set QT_QUICK_BACKEND=software

**Comment by YBPe** [comment]
How do I do that please?

**Comment by Jojo-Schmitz** [comment]
Exactly as written above. And before starting MuseScore. Both from a commandline

**Comment by YBPe** [comment]
I did it now, but the problem is'nt solved, still parts in Arabic. By the way, I heard it from more people using MuseScore in Hebrew.

**Comment by mercuree** [comment]
@YBPe 
Try the following:
1. WIN + R and type cmd, then press ENTER
2. copy/paste:
set QT_QUICK_BACKEND=software&& "C:\Program Files\MuseScore 4\bin\MuseScore4.exe"
then press ENTER

Let us know if it helps

**Comment by YBPe** [comment]
Thank you so much! It's working!

**Comment by YBPe** [comment]
Oh, there still a problem, the language works well only after I do the set QT_QUICK_BACKEND=software&& "C:\Program Files\MuseScore 4\bin\MuseScore4.exe" and afterwards when I open the software from my desktop it's back to the Arabic.

**Comment by Jojo-Schmitz** [comment]
Yes, you'd need that workaround again, until the bug got fixed

**Comment by Jojo-Schmitz** [comment]
You'll get notified, via this issue...

**Comment by cbjeukendrup** [comment]
Duplicate of https://github.com/musescore/MuseScore/issues/24334 and https://github.com/musescore/MuseScore/issues/24287.

@YBPe There's another workaround you can try: 
instead of 
```
set QT_QUICK_BACKEND=software&& "C:\Program Files\MuseScore 4\bin\MuseScore4.exe"
```
do
```
"C:\Program Files\MuseScore 4\bin\MuseScore4.exe" -fontengine=directwrite
```

Would be fixed by https://github.com/musescore/MuseScore/pull/23925, but we cannot use that fix yet because it has negative side effects that would need to be prevented first.

**Comment by YBPe** [comment]
The second option ""C:\Program Files\MuseScore 4\bin\MuseScore4.exe" -fontengine=directwrite" does'nt work.

**Comment by cbjeukendrup** [comment]
Hm, then the internal Qt.conf apparently takes precedence over command line options. 
You could try the builds from https://github.com/musescore/MuseScore/pull/23925, see [How to download test builds from pull requests](https://github.com/musescore/MuseScore/wiki/Downloading-and-running-test-builds#downloading-builds-from-pull-requests). Someone else reported that that PR fixed it for them; if it doesn't work for you, that would be a bit worrying.

**Comment by Jojo-Schmitz** [comment]
isn't it rather
```
"C:\Program Files\MuseScore 4\bin\MuseScore4.exe" -platform fontengine=directwrite
```

**Comment by cbjeukendrup** [comment]
Hm... or even 
```
"C:\Program Files\MuseScore 4\bin\MuseScore4.exe" -platform windows:fontengine=freetype
```

**Comment by YBPe** [comment]
The second one ""C:\Program Files\MuseScore 4\bin\MuseScore4.exe" -platform windows:fontengine=freetype" does'nt work. The previous one: "C:\Program Files\MuseScore 4\bin\MuseScore4.exe" -platform fontengine=directwrite shows me this:
![MuseScore language bug 3](https://github.com/user-attachments/assets/4dea3c97-6bd9-4ab9-a788-a4c1cf59ec59)

**Comment by cbjeukendrup** [comment]
Okay sorry this is a bit messy but with the last option I meant
```

"C:\Program Files\MuseScore 4\bin\MuseScore4.exe" -platform windows:fontengine=directwrite
```
(I accidentally copied `freetype` from the Qt docs)

**Comment by YBPe** [comment]
It does'nt work either

**Comment by mercuree** [comment]
This works for me:
``` 
"C:\Program Files\MuseScore 4\bin\MuseScore4.exe" --platform windows:fontengine=directwrite
```
However it opens Score tab by default

**Comment by YBPe** [comment]
I tried it, and this text appeared: C:\Program' is not recognized as an internal or external command,
operable program or batch file.

**Comment by mercuree** [comment]
@YBPe right click on Musescore desktop shortcut  - > Properties  
and there you can add `--platform windows:fontengine=directwrite`
This is temporary solution until Musescore is released

**Comment by YBPe** [comment]
Where in properties do I add it?

**Comment by mercuree** [comment]
@YBPe https://www.digitalcitizen.life/shortcut-arguments-parameters-windows/
This is all offtopic. I think I'm done here. I hope I helped you.

**Comment by YBPe** [comment]
Thanks! It was indeed  a major help!
